src/index.html

```diff
@@ -9,6 +9,39 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="header__logo" href="#home">
+        <img 
+          class="logo" 
+          src="images/logo.png" 
+          alt="logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__ul">
+          <li class="nav__li"><a class="nav__a is-active" href="#">Apple</a></li>
+
+          <li class="nav__li"><a class="nav__a" href="#">Samsung</a></li>
```

Comment:
```suggestion
          <li class="nav__li">
            <a class="nav__a" href="#">Samsung</a>
          </li>
```

it would be better to format your code this way